client/src/modules/reports/deprecated-modals/accounts_chart.modal.html

```diff
@@ -5,15 +5,15 @@
   </ol>
 </div>
 
-<form name="ConfigForm" bh-submit="ReportConfigCtrl.generate()" novalidate>
+<form name="ConfigForm" bh-submit="ReportConfigCtrl.generate()" bh-form-defaults novalidate>
```

Comment:
all these files which are in the `deprecated-modals` (old `modals`) are not used, and they must be removed, they are still there because this a first implementation of the new design of reports.